homeassistant/components/blebox/cover.py

```diff
@@ -31,16 +31,16 @@
 
 BLEBOX_TO_HASS_COVER_STATES = {
     None: None,
-    0: STATE_CLOSING,  # moving down
-    1: STATE_OPENING,  # moving up
-    2: STATE_OPEN,  # manually stopped
-    3: STATE_CLOSED,  # lower limit
-    4: STATE_OPEN,  # upper limit / open
-    # gateController
-    5: STATE_OPEN,  # overload
-    6: STATE_OPEN,  # motor failure
-    # 7 is not used
-    8: STATE_OPEN,  # safety stop
+    # all blebox covers
+    blebox_uniapi.cover.BleboxCoverState.MOVING_DOWN: STATE_CLOSING,
```

Comment:
Can you import `BleboxCoverState`?
`from blebox_uniapi.cover import BleboxCoverState`

```suggestion
    BleboxCoverState.MOVING_DOWN: STATE_CLOSING,
```